courses/free-courses-en.md

```diff
@@ -372,6 +378,7 @@
 * [Java for Mobile Devices - Introducing Codename One](https://codenameone.teachable.com/p/java-for-mobile-devices-introducing-codename-one)
 * [Java Online Training \| Edureka](https://www.youtube.com/watch?v=hBh_CC5y8-s) - (Edureka)
 * [Java Programming](https://testautomationu.applitools.com/java-programming-course/) - Angie Jones (Applitools)
+* [Java Programming](https://java-programming.mooc.fi) - (University of Helsinki)
```

Comment:
could you remove this?

You also submit it at:
- #5568

Reply:
Do the same https://github.com/EbookFoundation/free-programming-books/pull/5569#discussion_r718393858 with this.

File: https://github.com/Nandan-18/free-programming-books/blob/my-branch3/courses/free-courses-en.md